workflow rules docs + pricing note changes on existing features (#98)
team-plain/docs: workflow-rules.mdx

---
title: 'Workflow Rules'
---

<Frame>![Workflow Rules](/public/images/workflow-rules.png)</Frame>

<br/>
<Info>
Workflow Rules are available on the [Grow pricing plan](https://www.plain.com/pricing) and above.

If you are interested in trying this feature, please reach out to us at sales@plain.com or via our shared Slack channel.

</Info>

You can use workflow rules to help you automate your team's processes.

Some common use-cases include:

> - Assign someone to handle all inquiries from a given company or for your Enterprise tier

Reviewer: ```suggestion
- Assign someone to handle all support requests  from a given company or tier (e.g. Enterprise)
```

> - Assign a higher priority on a thread if it has a given label
> - Assign a specific label on a thread if it contains certain keywords ([Plain AI](/plain-ai) can also add labels for you)

Reviewer: ```suggestion
- Set a threads' priority if it includes certain key phrases
- Assign threads based on the used support email (e.g. Give all security@ emails to Jane)
```

To manage your workflow rules go to **Settings** → **Workflow Rules**.

When building a new rule, you can choose among a set of **conditions** which result in one or more **actions**.

There are a couple of conditions you can choose from, based on the thread's attributes:

- a given label is applied
- is from a given company
- is from a given slack channel
- has a specific tier (ie Enterprise)
- is for a specific support address (ie sales@yourcompany.com)
- contains a certain keyword (ie vulnerability)

> When the chosen condition matches a thread, you can perform one of the following actions:

Reviewer: ```suggestion
When the chosen condition matches a thread, you can perform any of the following actions:
```
